src/main/java/com/example/LoyaltyCard.java

```diff
@@ -0,0 +1,36 @@
+package com.example;
+
+import static com.google.common.base.Preconditions;
+
+
+/**
+ * Each customer has his own card that serves as an additional identifier.
```

Comment:
`his own` — this is a tautology.

`additional` — in addition to what? What kind of identifier does a customer already have?